coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/parser/CoralParseDriver.java

```diff
@@ -29,6 +42,17 @@ public ASTNode parse(String command) throws ParseException {
     HiveLexerCoral lexer = new HiveLexerCoral(new ANTLRNoCaseStringStream(command));
     TokenRewriteStream tokens = new TokenRewriteStream(lexer);
     HiveParser parser = new HiveParser(tokens);
+    HiveConf hiveConf = new HiveConf();
```

Comment:
why not pass the hiveConf from `HiveToRelConverter` to `CoralParseDriver`? and then the following if-else logic can be performed in the  `CoralParseDriver`'s constructor.

Reply:
Not sure what advantage we would gather there. If we want to expose the hiveConf from user, we would have to extend [HiveMetastoreClient.java](https://github.com/linkedin/coral/blob/master/coral-common/src/main/java/com/linkedin/coral/common/HiveMetastoreClient.java#L14) to have an additional function to getConf and pass in the conf used in the creation of HiveMetastoreClient all the way down to CoralParseDriver and I am not sure if that fix is relevant to the issue being fixed in this patch.